include/deal.II/grid/intergrid_map.h

```diff
@@ -98,9 +96,8 @@ DEAL_II_NAMESPACE_OPEN
  *   typename DoFHandler<dim>::cell_iterator cell = dof_handler_1.begin(),
  *                                           endc = dof_handler_1.end();
  *   for (; cell!=endc; ++cell)
- *                    // now do something with the cell of dof_handler_2
- *                    // corresponding to @p cell (which is one of
- *                    // dof_handler_1
+ *     // now do something with the cell of dof_handler_2 corresponding to
+ *     // @p cell (which is one of dof_handler_1's cells)
```

Comment:
On a different note, since you're here: this is part of a `@verbatim` environment, so `@p cell` is not formatted but just left alone. May want to just use "'cell'" here.

`@verbatim` environments are not formatted and colored as code, so this shows up less nice than could be. If you don't feel like addressing both these things (and I'm not going to make you), feel free to open a github issue for this.